SPECS/kernel/config

```diff
@@ -1,24 +1,24 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/x86_64 6.6.22.1 Kernel Configuration
+# Linux/x86 6.6.29.1 Kernel Configuration
 #
-CONFIG_CC_VERSION_TEXT="gcc (GCC) 13.2.0"
+CONFIG_CC_VERSION_TEXT="gcc (GCC) 11.2.0"
 CONFIG_CC_IS_GCC=y
-CONFIG_GCC_VERSION=130200
+CONFIG_GCC_VERSION=110200
```

Comment:
this is a detail of the toolchain you ran make on; we can safely revert this section to its original 3.0-dev values.